### DISCUSSION TITLE
Strobe Lights

### DISCUSSION THREAD

**Discussion by mreza1st** [flybywiresim/aircraft#3250] [Ideas]
Hi all,
Thanks again and again for continuing to do a great job with A320NEO. I was wondering if the strobe lights can be modified so they can be visible through the Wingviews and where they actually are on the winglet. In real life you can certainly see them. Please see the youtube link below at 1:40min
https://www.youtube.com/watch?v=TFc50SwEW8I
Thanks,

**Comment by mreza1st** [comment]
Is no one interested in this?

**Reply by hotshotp** [reply]
please be patient, it is being worked on
